vendor/itemproperties.lua

```diff
@@ -208,6 +208,10 @@ function Addon:GetItemProperties(arg1, arg2)
     -- Import the tooltip text as item properties for custom rules.
     item.TooltipLeft = self:ImportTooltipTextLeft(tooltip, bag, slot)
     item.TooltipRight = self:ImportTooltipTextRight(tooltip, bag, slot)
+    item.TooltipLeftColor = self:ImportTooltipColorLeft(tooltip, bag, slot)
+    item.TooltipRightColor = self:ImportTooltipColorRight(tooltip, bag, slot)
```

Comment:
Colors don't need to be imported and holding up memory in every item object. We only care about RedText. No need to import them, and if we did, we'd want a function to scan them for a particular color with a set of well defined colors (I'm not saying to do that, just pointing out that this change is incomplete regardless.)